jbt-it/neomdb
# #188 client fix login feedback error

Removed the deauthenticate-function-call from the login function.
It is not necessary to deauthenticate the user if the login was wrong. Because of the state-change that triggers when the deauthenticate-fucntion is called, the view reloads. This was the reason why no error was displayed, when a user input wrong credentials.

How to test:
1. Try to login in with wrong credentials
2. An error message should be displayed below the password input field
